packages/docs/docs/tutorial/integrate-sdk/android.mdx

```diff
@@ -7,27 +7,26 @@ import TabItem from '@theme/TabItem';
 
 # Integrate `io.logto.sdk:android`
 
-<Step
-  title="Install SDK"
-  subtitle="Install Logto Android SDK with Gradle"
-  index={0}
-  activeIndex={props.activeStepIndex}
-  invalidIndex={props.invalidStepIndex}
-  onNext={() => props.onNext(1)}
->
-
 ## Step1: Install Logto Android SDK with Gradle
 
 ### Prerequisite
-* Minimal Android SDK: Level 24 (TBD)
+* Minimal Android SDK: Level 24
+
+Add the `mavenCentral()` repository to your Gradle project build file:
```

Comment:
Do we need to specify which file is the `Gradle project build file`, in case there is a noob Android programmer, like me? ;-)

Reply:
Maybe we don't. There is always a build file in the project, something like `package.json` to `Node` users.